client/coral-admin/src/actions/users.js

```diff
@@ -1,14 +1,13 @@
+import coralApi from '../../../coral-framework/helpers/response';
 
 /**
  * Action disptacher related to users
  */
-//
-// export const banUser = (status, author_id) => (dispatch) => {
-//   dispatch({type: 'USER_STATUS_UPDATE', author_id, status});
-// };
-export const banUser = (status, userId, commentId) => {
+// change status of a user
+export const userStatusUpdate = (status, userId, commentId) => {
   return dispatch => {
-    dispatch({type: 'USER_BAN', status, userId, commentId});
-    dispatch({type: 'COMMENTS_MODERATION_QUEUE_FETCH'});
+    return coralApi(`/users/${userId}/status`, {method: 'POST', body: {status: status, comment_id: commentId}})
+      .then(res => dispatch({type: 'USER_BAN_SUCESS', res}))
+      .catch(error => dispatch({type: 'USER_BAN_FAILED', error}));
```

Comment:
Please use the postfixes to monitor the request. `_FAILED` should be `_FAILURE`

I.E
```js
export const FETCH_SIGNUP_REQUEST = 'FETCH_SIGNUP_REQUEST';
export const FETCH_SIGNUP_FAILURE = 'FETCH_SIGNUP_FAILURE';
export const FETCH_SIGNUP_SUCCESS = 'FETCH_SIGNUP_SUCCESS';
```